docs/website/docs/general-usage/credentials/configuration.md

```diff
@@ -0,0 +1,422 @@
+---
+title: Secrets and Configs
+description: Overview secrets and configs
+keywords: [credentials, secrets.toml, secrets, config, configuration, environment
+      variables]
+---
+
+# Secrets and Configs
+
+## General Usage and an Example
+
+This is what we had in mind, when designing:
```

Comment:
Ok, if we want to keep this part it is at least best to title it with something like "Design Principles"